interface-definitions/vpn_ipsec.xml.in

```diff
@@ -1244,6 +1244,31 @@
                     <children>
                       #include <include/ipsec/bind.xml.i>
                       #include <include/ipsec/esp-group.xml.i>
+                      <leafNode name="traffic-selectors">
```

Comment:
As this is not a multi node and it can be either ipv4, ipv6 or both I would use singular for the node name.

@dmbaturin what's your opinion on this?

Reply:
traffic-selectors means many prefixes
Even if it set to IPv4 => 0.0.0.0/0, the remote site could send several prefixes. As and local site as well.

We probably extend it in the future to 

x.x.x.x/x
x.x.y.y/x[gre]

Reply:
Speaking about multi-nodes...

Why not make this a multi-node with only `ipv4` and `ipv6` values? And activate both if the node is not presented in the config?

Reply:
> Why not make this a multi-node with only `ipv4` and `ipv6` values? And activate both if the node is not presented in the config?

It doesn't make sense now.
By default, without a node, it uses the default value `both`
If we agree, I'm ready to replace this with `/multi`.
Just let me know.

Reply:
> Why not make this a multi-node with only ipv4 and ipv6 values? And activate both if the node is not presented in the config?

This is the wrong way. If this should be a multi node, a migrator would be needed to change CLI to add both ipv4 and ipv6 options and not silently setting them under the hood. It should be obvious to the user. I kind of like the idea of a multi node, but it requires additional effort for the migrator and can not be backported to 1.4 as the CLI is fixed.